deploy/charts/venafi-kubernetes-agent/tests/deployment_test.yaml

```diff
@@ -0,0 +1,76 @@
+suite: test deployment
+templates:
+  - deployment.yaml
+
+tests:
+  # Basic checks on deployment
+  - it: templates as expected
+    set:
+      image.tag: latest
+      config.clientId: "00000000-0000-0000-0000-000000000000"
+    template: deployment.yaml
+    asserts:
+      - isKind:
+          of: Deployment
+      # Validate name matches
+      - matchRegex:
+          path: metadata.name
+          pattern: ^venafi-kubernetes-agent-*
+      # Check is latest is set as tag that it uses that tag
+      - equal:
+          path: spec.template.spec.containers[0].image
+          value: quay.io/jetstack/preflight:latest
```

Comment:
Are we going to change the build process for this to a new repo/ image?

Reply:
Unsure right now, I just want to focus on the easiest possible path and for me, use the same image.
The name SHOULD ABSOLUTELY be changed though before release IMO.